HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # Rocksdb Change Log
 ## Unreleased
+### Public API Change
+* File I/O listener is extended to cover more I/O operations. Now class `EventListener` in listener.h contains new callback functions: `OnFileFlushFinish()`, `OnFileSyncFinish()`, `OnFileRangeSyncFinish()`, `OnFileTruncateFinish()`, and ``OnFileCloseFinish()``.
+
```

Comment:
This can be combined with the `Public API Change` section.

Reply:
Ah I saw there are two of them below, so I thought it was one-header-per-PR basis. Updated.